README.md

```diff
@@ -1 +1,203 @@
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [LEADERBOARD] <a name="about-project"></a>
+
+**[Leaderboard]** is a project to display diffrent scores of participants in a contest.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://html.com/">HTML</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://nodejs.org/">Node js</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>API</summary>
+  <ul>
+    <li><a href="https://www.postman.com/">Postman</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+
+
+- **[The application has form inputs to collect score data and display it on a scoreboard]**
+- **[the refresh button checks for new entries in the Api and displays it on the screen]**
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+
+- [Live Demo Link]()
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+
+```
+ git installed
+```
+
+
+### Setup
+
+Clone this repository to your desired folder:
+
+
+```sh
+  cd leaderboard
+  git clone https://github.com/Obimbo07/leaderboard.git
+```
+
+
+### Install
+
+Install this project with:
+
+
+
+```sh
+  cd leaderboard
+  git install
+```
+--->
+
+### Usage
+
+To run the project, execute the following command:
+
+
+
+```sh
+  npm run start
+```
+
+
+### Run tests
+
+To run tests, run the following command:
+
+
+```sh
+ npm test
+```
+--->
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+> Mention all of the collaborators of this project.
+
+👤 **Author1**
+
+- GitHub: [@Obimbo07](https://github.com/Obimbo07)
+- Twitter: [@Austine](https://twitter.com/austsine)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/austin-obimbo-9a613623a/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+
+- [ ] **[The project can be displaying real-time scoreboard data from diffrent contest]**
+- [ ] **[A live link to the performance score and profile bio of the participant]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).

```

Comment:
- [ ] Please insert the valid link for the issues page of this specific project for a professional README file. 🙏